lightwallet-client-lib/src/main/java/co/electriccoin/lightwallet/client/internal/BlockingLightWalletClientImpl.kt

```diff
@@ -126,7 +126,7 @@ internal class BlockingLightWalletClientImpl private constructor(
         }
 
         val result = requireChannel().createStub().getAddressUtxos(
-            Service.GetAddressUtxosArg.newBuilder().setAddress(tAddress)
+            Service.GetAddressUtxosArg.newBuilder().setAddresses(Constants.TRANSPARENT_ADDRESS_INDEX, tAddress)
```

Comment:
I think just putting zero here is fine. If I'm not mistaken, the builder lets you request UTXOs for many addresses at once and the -somewhat misfortunate- builder API is asking of an index for the request. This index has nothing to do with the t-address index. In fact the `DiversifierIndex` of the t address you might be requesting will probably not match the index you need to build this query.
```suggestion
            Service.GetAddressUtxosArg.newBuilder().setAddresses(0, tAddress)
```

Reply:
Ok, understand. Thanks